examples/sdk-examples/package.json

```diff
@@ -84,7 +84,7 @@
     "dependencies": {
         "@gooddata/api-client-bear": "^8.2.0-alpha.16",
         "@gooddata/api-model-bear": "^8.2.0-alpha.16",
-        "@gooddata/goodstrap": "^70.0.2",
+        "@gooddata/goodstrap": "~70.1.0",
```

Comment:
Please use carets. There is a `--caret` switch in `rush add`.

Reply:
Thanks, I'll change it (: